### PR TITLE
Handle errors when there is no configured data

### PR DESCRIPTION
**Pull request by evmiguel**
On a fresh database with no active runs, there were errors in rendering the "Configure Active Runs" page. This can be implemented better, but @dawnwages needs this to get up and running.

### CODE REVIEW

File: server/services/UsersService.js [hunk 1]
```diff
@@ -249,7 +249,7 @@ async function saveUserAndRoles(options) {
             }
```
Comment (evmiguel) [inline server/services/UsersService.js:249]: Lint fix

**Comment by rmeritz** [comment]
@evmiguel - I think since @dawnwages isn't working on this now. I think we should not add anything to the code with "this is a hack". I think it would be better to just have @dawnwages do a better fix. 

What do you think the better solution is?

**Comment by evmiguel** [comment]
> @evmiguel - I think since @dawnwages isn't working on this now. I think we should not add anything to the code with "this is a hack". I think it would be better to just have @dawnwages (or one of us) do a better fix.
> 
> What do you think the better solution is?

In the [initial state](https://github.com/w3c/aria-at-app/pull/213/files#diff-bf6af43baaa40d6e4b093436ee6862061595542cf09f26c00a3ae2b1f0a96a51R78) for `runTechnologyRows`, it should just be an empty array instead of an array with one empty object.

The empty object yields checking for the emptiness of the object with `Object.keys(this.state.runTechnologyRows[0]).length`, which could be cleaner if just checking for length. Then, I could remove [this line](https://github.com/w3c/aria-at-app/pull/213/files#diff-bf6af43baaa40d6e4b093436ee6862061595542cf09f26c00a3ae2b1f0a96a51R572) and just check for the length of the `runTechnologyRows` in [this line](https://github.com/w3c/aria-at-app/pull/213/files#diff-bf6af43baaa40d6e4b093436ee6862061595542cf09f26c00a3ae2b1f0a96a51R653).

I can fix it, since it will be quick.

File: client/components/App/App.css [hunk 2]
```diff
@@ -52,9 +52,10 @@ nav.navbar {
     background: #cfcece;
```
Comment (evmiguel) [inline client/components/App/App.css:52]: Lint fix

File: client/components/ConfigureActiveRuns/ConfigureActiveRuns.css [hunk 3]
```diff
@@ -6,19 +6,20 @@
 }
```
Comment (evmiguel) [inline client/components/ConfigureActiveRuns/ConfigureActiveRuns.css:6]: Lint fix

File: client/components/ConfigureActiveRuns/index.jsx [hunk 4]
```diff
@@ -632,45 +698,22 @@ class ConfigureActiveRuns extends Component {
...
-                                                        .undoDeleteTechnologyRow
-                                                }
-                                                editable={runTech.editable}
-                                                deleted={runTech.deleted}
-                                            />
-                                        );
-                                    }
-                                )}
-                        </tbody>
+                        {technologyRows.length ? (
```
Comment (evmiguel) [inline client/components/ConfigureActiveRuns/index.jsx:701]: Hack fixed

File: client/components/ConfigureTechnologyRow/ConfigureTechnologyRow.css [hunk 5]
```diff
@@ -1,23 +1,24 @@
-
```
Comment (evmiguel) [inline client/components/ConfigureTechnologyRow/ConfigureTechnologyRow.css:1]: Lint fix

File: client/components/ConfigureTechnologyRow/index.jsx [hunk 6]
```diff
@@ -138,7 +138,7 @@ class ConfigureTechnologyRow extends Component {
                 <td>{runTechnologies.browser_version}</td>
```
Comment (evmiguel) [inline client/components/ConfigureTechnologyRow/index.jsx:138]: Lint fix

File: client/components/TestQueue/TestQueue.css [hunk 7]
```diff
@@ -1,4 +1,4 @@
-main.container-fluid table .row,
```
Comment (evmiguel) [inline client/components/TestQueue/TestQueue.css:1]: Lint fix

File: client/components/TestQueueRun/TestQueueRun.css [hunk 8]
```diff
@@ -28,7 +28,7 @@ li {
 }
```
Comment (evmiguel) [inline client/components/TestQueueRun/TestQueueRun.css:28]: Lint fix

File: client/components/TestRun/index.jsx [hunk 9]
```diff
@@ -440,7 +440,11 @@ class TestRun extends Component {
```
Comment (evmiguel) [inline client/components/TestRun/index.jsx:440]: Lint fix

File: client/components/TestRun/TestRun.css [hunk 10]
```diff
@@ -165,8 +165,7 @@ button.test-navigator-toggle.show:after {
     padding: 0;
```
Comment (evmiguel) [inline client/components/TestRun/TestRun.css:165]: Lint fix

File: client/components/TestQueueRun/index.jsx [hunk 11]
```diff
@@ -1,7 +1,11 @@
 import React, { Component, Fragment } from 'react';
```
Comment (evmiguel) [inline client/components/TestQueueRun/index.jsx:1]: Lint fix

**Comment by evmiguel** [comment]
@rmeritz all set!

**Pull request merged**